presto-native-execution/presto_cpp/main/types/PrestoToVeloxExpr.cpp

```diff
@@ -66,11 +86,15 @@ std::string mapScalarFunction(const std::string& name) {
 }
 
 std::string mapAggregateOrWindowFunction(const std::string& name) {
+  static const std::string prestoDefaultNamespacePrefix =
+      SystemConfig::instance()->prestoDefaultNamespacePrefix();
   static const std::unordered_map<std::string, std::string> kFunctionNames = {
-      {"presto.default.$internal$max_data_size_for_stats",
-       "presto.default.max_data_size_for_stats"},
-      {"presto.default.$internal$sum_data_size_for_stats",
-       "presto.default.sum_data_size_for_stats"},
+      {fmt::format(
```

Comment:
Write an inline function for adding the defaultNamespacePrefix to the function name and use it everywhere.